config/locales/missing_endpoints.fr.yml

```
@@ -0,0 +1,32 @@
+fr:
+  missing_endpoints:
+    /v3/agence_bio/certifications_bio:
+      title: 'Certification en BIO'
+      description: 'Connaître les certifications biologiques en cours d’une entreprise ou d’une association.'
+      disclaimer: &generic_disclaimer |+
+        <div class="fr-alert fr-alert--error">
+          <p class="fr-alert__title">
+            Ce endpoint n'est actuellement pas implémenté sur la v3
+          </p>
+          <p>
+            Les développements sont en cours et devrait être terminés dans le semaines qui suivent.
```

```suggestion
             Les développements sont en cours et devraient être terminés dans les mois qui suivent.
```

Pas obligé de mettre "mois" cela dit x^)